Font sheets for spriters (#3436)
new-frontiers-14/frontier-station-14: Tools/_NF/font_sheets/attributions.txt

Reviewer: Should this be a YAML file instead of a .TXT?

Author: Linter complains

Author: If its yml

Reviewer: because your source is poorly formatted

Author: Impossible.
It's clearly because linter is a shitter and has something against me.

﻿- files: ["font_sheet_2x3.png"]
  license: "CC-BY-4.0"

>   copyright: "Based on 2x3 font by dustylens (discord/github) and 3x3 pixel font by Akaki Hojo; spliced together with minor additions by erhardsteinhauer (discord/github)"
>   source: "https://akakihojo.itch.io/3x3-pixel-alphabet-numbers-font, https://github.com/new-frontiers-14/frontier-station-14/tree/master/Resources/Textures/Structures/Wallmounts/posters.rsi"

Reviewer: ```suggestion
  copyright: "Based on 2x3 font by dustylens (github) (see Structures/Wallmounts/posters.rsi) and 3x3 pixel font by Akaki Hojo; spliced together with minor additions by erhardsteinhauer (github)"
  source: "https://akakihojo.itch.io/3x3-pixel-alphabet-numbers-font"
```
:+1:
also just pick one for discord/github

- files: ["font_sheet_3x3.png"]
  license: "CC-BY-4.0"

>   copyright: "Based on 3x3 pixel fonts by Akaki Hojo and Tony Pai; spliced together with minor additions by erhardsteinhauer (discord/github)"
>   source: "https://akakihojo.itch.io/3x3-pixel-alphabet-numbers-font, https://tonypai.itch.io/3x3-pixel-font"

Reviewer: ```suggestion
  copyright: "Based on 3x3 pixel fonts by Akaki Hojo (at https://akakihojo.itch.io/3x3-pixel-alphabet-numbers-font) and Tony Pai; spliced together with minor additions by erhardsteinhauer (github)"
  source: "https://tonypai.itch.io/3x3-pixel-font"
```
:+1:

- files: ["font_sheet_3x3alt.png"]

>   license: "CC-BY-4.0"

Reviewer: ```suggestion
  license: "CC-BY-SA-3.0"
```
original terms from the link provided, though you could maybe upgrade to 4

  copyright: "Based on 3x3 pixel font by Håvar Henriksen (havarh); edited by erhardsteinhauer (discord/github)"
  source: "https://fontstruct.com/fontstructions/show/492714/tiny3"

- files: ["font_sheet_3x5.png"]

>   license: "CC-BY-3.0"

Reviewer: ```suggestion
  license: "CC-BY-SA-3.0"
```
tgstation resource, should be CC-BY-SA-3.0, and any relicense should require the self-attribution bit.
